lms/templates/main_django.html

```diff
@@ -28,14 +28,14 @@
   <div class="window-wrap" dir="${static.dir_rtl()}">
     <a class="nav-skip" href="{% block nav_skip %}#content{% endblock %}">{% trans "Skip to main content" %}</a>
     {% with course=request.course %}
-      {% include "header.html" %}
+      {% include "header.html"|microsite_template_path %}
```

Comment:
@asadiqbal08 -- okay, let's go ahead and make the necessary changes in the edx-microsites repository.  Please log a WL ticket to track the job and add it to the current sprint.